sirepo/package_data/static/js/activait.js

```diff
@@ -282,11 +282,32 @@ SIREPO.app.controller('AnalysisController', function (appState, activaitService,
     buildSubplots();
 });
 
-SIREPO.app.controller('DataController', function (activaitService, appState) {
+SIREPO.app.controller('DataController', function (activaitService, appState, $scope) {
     const self = this;
     self.activaitService = activaitService;
-    self.showImageViewer = () => activaitService.isImageData()
-        && appState.models.columnInfo.header;
+    const c = appState.models.columnInfo;
+
+    const inOut = (inputOutput) => {
+        let res = true;
+        ['input', 'output'].forEach(x => {
+           if (!inputOutput.includes(x)){
+               res = false;
+           }
+        });
+        return res;
+    };
```

Comment:
Now that I understand the point of this function, it could be further "reduced":
```javascript
const inOut = inputOutput => ['input', 'output'] .map(x => inputOutput.includes(x)).reduce((p, c) => p && c);
```
Not a big deal though. I might clarify the name of the function (hasInputAndOutput or something)